### ISSUE TITLE
/entrypoint.sh change

### ISSUE THREAD

**Issue by zdykstra** [docker-suite/alpine-runit#1]
After rebuilding a few internal containers on the most recent `dsuite/alpine-runit` image, I wasn't able to start any container.  https://github.com/docker-suite/alpine-runit/commit/3c4395825980d6cb5b688667f97dde0dad1bdace#diff-6710c73f0c6b66edade17b5c6d99f6b1083f1e22ae612ae294cff9748a246d8fR35 changed the entrypoint to include `tini --` - was that intentional? I've resolved the issue in my images by flipping the entrypoint back to what it was, so that runit gets the correct command.

**Comment by zdykstra** [comment]
Since `runit` is just a shell script, I've changed my image to `ENTRYPOINT ["tini", "--", "/entrypoint.sh"]` so that pid1 is `tini` instead of bash.

`/entrypoint.sh` as shipped in the container exec's runit with all of the args passed to it.

```
/ # tail -2 /entrypoint.sh
### Execute script with arguments
exec runit "${@}"
```

**Comment by hexosse** [comment]
My mistake.
However, I removed tini from the entrypoint as it is included in Docker itself since 1.13 and can be used with docker --init argument
